lib/LanguageServerRename/Adapter/Worse/RenameLocationsProvider.php

```diff
@@ -9,6 +9,9 @@
 use Phpactor\TextDocument\ByteOffset;
 use Phpactor\TextDocument\TextDocument;
 
+/**
+ * @deprecated The renamers should stream LocatedTextEdit instances.
+ */
 class RenameLocationsProvider
```

Comment:
these classes a redundant with the new refactoring, but I'm too tired to factor them out at the moment.

Reply:
Did you change the reference finder to return the definition?

Reply:
No, I introduced a ReferenceFinder which uses both a definition locator and a reference finder, so kindof.